### PR TITLE
fix: vitesse-drak-soft terminal.foreground

### LINKED ISSUE #14: Bug: terminal background
### Describe the bug

WeChatd0cdc386f5165f9a9551aa072477a316
WeChatf9f9666a940215905eea0d4de4400e34

when you paste something into the terminal , this bug will happen.

### Reproduction

none

### System Info

```Shell
terminal
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### PR DESCRIPTION
**Pull request by Simon-He95**
close: #14

WeChat31b621fb1cf87c79c4bac4728e5fe4ac
WeChat3651868c488eedeae75b02328227d450

### CODE REVIEW

**Comment by antfu** [comment]
I don't have the issue - how would removing opacity helps?

**Comment by Simon-He95** [comment]
Problem with 8 digits，but 6 is fine
   
 "terminal.foreground": "#d7caee"

 I think `#dbd7ca` is more closer origin `#dbd7caee`

image

**Comment by antfu** [comment]
This feels like a bug of your terminal not supporting alpha channel?

File: themes/vitesse-dark-soft.json [hunk 1]
```diff
@@ -127,7 +127,7 @@
     "panelTitle.activeForeground": "#dbd7caee",
     "panelTitle.inactiveForeground": "#959da5",
     "panelInput.border": "#2f363d",
-    "terminal.foreground": "#dbd7caee",
+    "terminal.foreground": "#dbd7ca",
```
Comment (antfu) [inline themes/vitesse-dark-soft.json:130]: You should update the src/theme.ts instead of .json, this fill is generated.

Comment (Simon-He95) [reply themes/vitesse-dark-soft.json:130]: I made adjustments

**Comment by Simon-He95** [comment]
> This feels like a bug of your terminal not supporting alpha channel?

I make a test and only individual 8 digits have this problem.
I think our terminal should be the same by ohmyzsh

**Comment by Simon-He95** [comment]
@antfu can you help me merge or Do you think I can modify it myself and send a separate vscode plugin？

**Comment by lhz960904** [comment]
I have same problem,  It seems that the transparency overlay is overlapping the text at om-my-zsh termainal。 currently fixed by overriding `terminal.foreground` in setting.json。e.g.
```json
{
  "workbench.colorCustomizations": {
      "terminal.foreground": "#dbd7ca" // replce "#dbd7cacc"
  }
}
```
only the dark-soft theme can be reproduced here, step: paste a words in vscode termainal

**Pull request closed without merging**